<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кривые второго порядка</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="curveList">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>200</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>600</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>9</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>620</y>
      <width>806</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="invariants">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>660</y>
      <width>806</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuRhbdst">
    <property name="title">
     <string>Кривые</string>
    </property>
    <addaction name="addCurveAction"/>
    <addaction name="separator"/>
    <addaction name="delCurveAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuRhbdst"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="addCurveAction">
   <property name="text">
    <string>Добавтиь кривую</string>
   </property>
  </action>
  <action name="delCurveAction">
   <property name="text">
    <string>Удалить текущую кривую</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGraphicsView</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
